objects/src/transaction/inputs.rs

```diff
@@ -134,6 +134,28 @@ impl TransactionInputs {
     }
 }
 
+impl Deserializable for TransactionInputs {
+    fn read_from<R: ByteReader>(source: &mut R) -> Result<Self, DeserializationError> {
+        let account = Account::read_from(source)?;
+        let account_seed = source.read()?;
+        let block_header = BlockHeader::read_from(source)?;
+        let block_chain = ChainMmr::read_from(source)?;
+        let input_notes = InputNotes::read_from(source)?;
+        Self::new(account, account_seed, block_header, block_chain, input_notes)
+            .map_err(|err| DeserializationError::InvalidValue(format!("{}", err)))
+    }
+}
+
+impl Serializable for TransactionInputs {
+    fn write_into<W: ByteWriter>(&self, target: &mut W) {
+        self.account.write_into(target);
+        self.account_seed.write_into(target);
+        self.block_header.write_into(target);
+        self.block_chain.write_into(target);
+        self.input_notes.write_into(target);
+    }
+}
```

Comment:
nit: we usually put `Serializable` implementation above `Deserializable`.

Reply:
Oh, I will swap them then